content/3_Silver/Prefix_Sums_2.mdx

```diff
@@ -102,11 +102,102 @@ export const problems = {
 
 <FocusProblem problem={problems.maxsum} />
 
+### Solution
+
+<Spoiler title="Solution">
+
+Consider the prefix sum array $p$. The subarray sum $a_j \dots a_{i-1}$ where $j\leq i$ is $p[i]-p[j]$, so we are looking for the maximum possible value of $p[i]-p[j]$ over $0 \leq i \leq j \leq N$. 
+
+For a fixed right bound $i$, the maximum subarray sum is
+
+$$$
```

Comment:
Why are there 3 `$`? Shouldn't 2 work?

Reply:
idk i just commited what the other ppl recommended.